README.md

```diff
@@ -32,10 +32,16 @@
 ## Table of Contents <!-- omit in TOC -->
 
 - [📖 Documentation](#-documentation)
-- [⚡ Supercharge your Meilisearch experience](#-supercharge-your-meilisearch-experience)
+- [💫 Supercharge your Meilisearch experience](#-supercharge-your-meilisearch-experience)
 - [🔧 Installation](#-installation)
 - [🚀 Getting started](#-getting-started)
+  - [Add documents](#add-documents----omit-in-toc---)
```

Comment:
There is an issue with the link of the sub-sections.

Remove `omit in TOC` in title and in TOC link I think if you want them to appear correctly